openmetadata-ui/src/main/resources/ui/src/components/common/DeleteWidget/DeleteWidgetModal.tsx

```diff
@@ -85,16 +85,15 @@ const DeleteWidgetModal = ({
   const DELETE_OPTION = useMemo(
     () => [
       {
-        title: `${t('label.delete')} ${entityTypeName} "${entityName}"`,
+        title: `${t('label.delete')} ${entityTypeName} ${entityName}`,
         description: (
```

Comment:
we don't want to remove the quotes from the `entityName`, since it is there to highlight the name and what we are deleing. Please restore this and the other places as well!